administrator/language/de-DE/plg_content_confirmconsent.ini

```
@@ -8,7 +8,11 @@ PLG_CONTENT_CONFIRMCONSENT="Inhalt - Datenschutzerklärung"
 PLG_CONTENT_CONFIRMCONSENT_CONSENTBOX_LABEL="Datenschutzhinweis"
 PLG_CONTENT_CONFIRMCONSENT_FIELD_ARTICLE_DESC="Beitrag aus der Liste auswählen oder einen neuen erstellen."
 PLG_CONTENT_CONFIRMCONSENT_FIELD_ARTICLE_LABEL="Datenschutzerklärung"
+PLG_CONTENT_CONFIRMCONSENT_FIELD_MENU_ITEM_LABEL="Datenschutz-Menüeintrag"
```

```suggestion
PLG_CONTENT_CONFIRMCONSENT_FIELD_MENU_ITEM_LABEL="Datenschutz-Menüpunkt"
```

Menüpunkt dann ist es bei allen strings gleich?

---

my fault